### PR TITLE
Create english readme

### LINKED ISSUE #8: Create English version of README

### PR DESCRIPTION
**Pull request by tyronedamasceno**
Create a version of README in english. 

closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
-# Bahea Calendar
+# Bahea Calendar :us:
+
+Bahea Calendar is a project that will add to your google calendar the schedule of your favorite soccer team games!
```
Comment (ivancrneto) [inline README.md]: "...that adds the standings of your favorite soccer team games to your google calendar!"

Sounds a bit better

File: README.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+
+## How to run
+
+You must have Python3 and pip installed on your computer.
+
+Also, you need to create a project to access the google calendar API. You can do it accessing the google's [developers console](https://console.developers.google.com/). When it's done, download the JSON file with your credentials and rename it with `credentials.json`
```
Comment (ivancrneto) [inline README.md]: You can do it *by* accessing

rename it ~with~ *to*

**Review by ivancrneto — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tyronedamasceno) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (tyronedamasceno) [reply README.md]: Done

**Review by ivancrneto — APPROVED** [review]
LGTM

**Pull request merged**